apis/opts/config/runtime_test.go

```diff
@@ -114,3 +154,35 @@ func TestRuntimeType(t *testing.T) {
 		}
 	}
 }
+
+func TestRuntimeString(t *testing.T) {
+	type fields struct {
+		values *map[string]types.Runtime
+	}
+
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "get string of Runtime",
+			fields: fields{
+				values: &(map[string]types.Runtime{
```

Comment:
Can you add a empty string and a nil values, like 
```
values: &(map[string]types.Runtime{}
values: &(map[string]types.Runtime{"": xxx}
```

Reply:
Thanks for your suggestion. I have already added the two test cases you suggested.